ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -28,11 +30,21 @@ class FaultInjector
 
   void faultsConfigCb(ow_faults::FaultsConfig& faults, uint32_t level);
 
+  enum Nominal {None=0};
+
+  enum ArmFaults {Hardware=1, TrajectoryGeneration=2, Collision=3, Estop=4, PositionLimit=5, TorqueLimit=6, VelocityLimit=7, NoForceData=8};
```

Comment:
Please spread these enums onto multiple lines for readability.

Reply:
done!